EasyPost/CreditCard.cs

```diff
@@ -0,0 +1,112 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using RestSharp;
+
+namespace EasyPost
+{
+    public class CreditCard : Resource
+    {
+        [JsonProperty("id")]
+        public string id { get; set; }
+        [JsonProperty("disabled_at")]
+        public string disabled_at { get; set; }
+        [JsonProperty("object")]
+        public string Object { get; set; }
+        [JsonProperty("name")]
+        public string name { get; set; }
+        [JsonProperty("last4")]
+        public string last4 { get; set; }
+        [JsonProperty("exp_month")]
+        public string exp_month { get; set; }
+        [JsonProperty("exp_year")]
+        public string exp_year { get; set; }
+        [JsonProperty("brand")]
+        public string brand { get; set; }
+
+        /// <summary>
+        ///     Payment method priority
+        /// </summary>
+        public enum Priority
```

Comment:
I'd prefer we kept variable names consistent with the other libs. `PrimaryOrSecondary`? Same goes for the param below.

Reply:
So `PrimaryOrSecondary.Secondary`?